plugins/org.wso2.developerstudio.eclipse.esb.libraries/.classpath

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
+	<classpathentry exported="true" kind="lib" path="lib/synapse-core_2.1.7.wso2v82.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/synapse-extensions_2.1.7.wso2v80.jar"/>
```

Comment:
the coreversion is wso2v82 but the extensions version is wso2v**80**, is this intentional?

Reply:
Yes, this change was intentional. We have done some validation fixes to the synapse core. Btw, we are in the process of releasing the synapse v82. After the release, we will add official synapse v82 jar (before merging this PR)